csharp/src/Apache.Arrow/Memory/LargeImportedAllocationOwner.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Apache.Arrow.Memory
+{
+    internal abstract class LargeImportedAllocationOwner:ILargeNativeAllocationOwner
```

Comment:
As above, we can remove this for now, it's only needed for the C Data Interface.